src/i18n/en.js

```diff
@@ -26,6 +26,13 @@ export default function loadEn() {
       'contact-page-media-alt':
         'Person using a magnifying glass to see the details of another guy.',
       'slack-org': 'Slack Organization',
+      close: 'Close',
+      'privacy-policy': `Effective date: October 03, 2018 {{lineBreak}}
```

Comment:
Do we really need `{{lineBreak}}` here? Maybe we can just do it with css?

Reply:
No idea either

Reply:
For titles is fine, I was just thinking about the line breaks.

Reply:
<img width="1680" alt="screen shot 2018-10-25 at 7 21 24 pm" src="https://user-images.githubusercontent.com/2995641/47505168-3a999e80-d88b-11e8-8328-5d71095bde4d.png">

@crysfel I don't think css option is working very well.

Reply:
Let's just go with the linebrakes and everything inside the i18n files, then we can iterate and come up with a better solutions, that maybe includes loading the content async, after all users won't open this modal very often